apps/escuelafrontend/src/pages/content.tsx

```diff
@@ -0,0 +1,26 @@
+import {useSession} from 'next-auth/react'
+import Link from 'next/link'
+
+export default function Content() {
+  const {data: session, status} = useSession()
+
+  if (status === 'authenticated') {
+    return (
+      <>
+        Signed in as {session.user && session.user.email} <br />
+        <p>secreate badass content!</p>
+      </>
+    )
+  }
```

Comment:
generally speaking I want to do this work in SSR or middleware and note rely on `useSession` which is async